SharpPcap/LibPcap/LibPcapSafeNativeMethods.Unix.cs

```diff
@@ -39,7 +41,80 @@ internal static class Unix
         //       This file is called $assembly_name.dll.config and is placed in the
         //       same directory as the assembly
         //       See http://www.mono-project.com/Interop_with_Native_Libraries#Library_Names
-        private const string PCAP_DLL = "libpcap";
+        private const string PCAP_DLL = "wpcap";
```

Comment:
Should this be npcap?

Reply:
Yes, for supporting .NET Framework that only runs on Windows

Reply:
Ahh so in the case where we don't have native library support this would let things work under winpcap or npcap + compatibility? I'm wondering if this should be npcap?

Reply:
npcap uses the dll name wpcap, when not in winpcap compatibility mode, the dll is still called wpcap, it's just nested in Npcap folder.

for that case the existing SetDllDirectory solves the problem